source/whats-new.txt

```diff
@@ -49,105 +49,105 @@ The following changes affect future versions of the driver:
-     Server manual entry.
+     collection.aggregate(Arrays.asList(
+                      new Document("$collStats",
+                          new Document("storageStats", new Document()))));
 
-.. fulfills JAVA-4845: (from Upcoming Breaking Changes) getstats and iscapped deprecated.
+  To determine whether a collection is a capped collection, access the value
+  of the ``storageStats.capped`` field returned by the preceding example
+  aggregation.
 
-   - The following network address-related methods are deprecated:
+  To learn more about the ``$collStats`` aggregation operator, see the
+  :manual:`$collStats (aggregation) </reference/operator/aggregation/collStats/>`
+  Server manual entry.
 
-      - ``getSocketAddress()`` and ``getSocketAddresses()`` methods from the `ServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/ServerAddress.html>`__ class
-      - ``getUnixSocketAddress()`` from the `UnixServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/UnixServerAddress.html>`__ class
+- The following network address-related methods are deprecated: (TODO: replace this item with https://github.com/mongodb/docs-java/pull/446/files)
 
-.. fulfills DOCSP-32942
+  - ``getSocketAddress()`` and ``getSocketAddresses()`` methods from the `ServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/ServerAddress.html>`__ class
+  - ``getUnixSocketAddress()`` from the `UnixServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/UnixServerAddress.html>`__ class
 
-      - The following Stream API methods and classes are deprecated and are
-        scheduled for removal in v5.0:
+- The following Stream API methods and classes are deprecated and are
+  scheduled for removal in v5.0:
 
-        - ``streamFactoryFactory()`` method from ``MongoClientSettings.Builder``
-        - ``getStreamFactoryFactory()`` method from ``MongoClientSettings``
-        - ``NettyStreamFactoryFactory`` class
-        - ``NettyStreamFactory`` class
-        - ``AsynchronousSocketChannelStreamFactory`` class
-        - ``AsynchronousSocketChannelStreamFactoryFactory`` class
-        - ``BufferProvider`` class
-        - ``SocketStreamFactory`` class
-        - ``Stream`` class
-        - ``StreamFactory`` class
-        - ``StreamFactoryFactory`` class
-        - ``TlsChannelStreamFactoryFactory`` class
+  - ``streamFactoryFactory()`` method from ``MongoClientSettings.Builder``
+  - ``getStreamFactoryFactory()`` method from ``MongoClientSettings``
+  - ``NettyStreamFactoryFactory`` class
+  - ``NettyStreamFactory`` class
+  - ``AsynchronousSocketChannelStreamFactory`` class
+  - ``AsynchronousSocketChannelStreamFactoryFactory`` class
+  - ``BufferProvider`` class
+  - ``SocketStreamFactory`` class
+  - ``Stream`` class
+  - ``StreamFactory`` class
+  - ``StreamFactoryFactory`` class
+  - ``TlsChannelStreamFactoryFactory`` class
 
-.. fulfills DOCSP-33033
+  To configure Netty by using a nondeprecated API, use the
```

Comment:
We should say here that the only ones being replaced with an alternative are the first three, which together let you configured Netty transport.  The others have no replacement, but don't need any.

Reply:
Just to confirm, the NettyStreamFactory (fourth in the list) isn't related?

Reply:
It's related, but there is no replacement for it.  It's not needed.